OPAL/tac/src/main/resources/org/opalj/tac/cg/android/AndroidCallbackList.txt

```diff
@@ -0,0 +1,181 @@
+android/accounts/OnAccountsUpdateListener
```

Comment:
What is the purpose of this file? I don't see it used anywhere?

Reply:
was kept for reference, i'll just delete it